test/collection_test.js

```diff
@@ -610,6 +610,49 @@ describe("Collection", () => {
           last_modified: remote.last_modified
         });
     });
+
+    it("should ignore resolved conflicts during sync", () => {
+      const remote = Object.assign({}, local, {
+        title: "blah",
+        last_modified: 42,
+      });
+      const conflict = {
+        type: "incoming",
+        local: local,
+        remote: remote,
+      };
+      const resolution = Object.assign({}, local, {
+        title: "resolved"
+      });
+      sandbox.stub(KintoClientCollection.prototype, "listRecords").returns(
+        Promise.resolve({
+          data: [
+            remote,
+          ],
+          next: () => {},
+          last_modified: "\"42\"",
+        }));
+      const syncResult = new SyncResultObject();
+      return articles.resolve(conflict, resolution)
+        .then(() => articles.pullChanges(syncResult))
+        .should.eventually.become({
+          ok: true,
+          lastModified: 42,
+          errors:    [],
+          created:   [],
+          published: [],
+          resolved:   [{
```

Comment:
Now that we have `void` type, there is nothing about this locally updated record at this staged. It is pending for publication, and will be listed in `published` when `pushChanges()` will be called.